data/advisories/LP-2020-01.md

```diff
@@ -4,7 +4,7 @@ title: "LP-2020-01: XSS vectors in laminas-api-tools/api-tools"
 date: '2020-04-01T16:30:00-05:00'
 ---
 
-# LP-2020-01: XSS vectors in laminas-api-tools/api-tools
+## LP-2020-01: XSS vectors in laminas-api-tools/api-tools
```

Comment:
Why does this need to be changed? It must be the first headline: https://getlaminas.org/security/advisory/LP-2020-01

Reply:
So looks like configuring `front_matter_title` to `''` will solve the problem